Tasks/src/StringArrayFilter.java

```diff
@@ -0,0 +1,96 @@
+import java.util.Scanner;
+
+public class StringArrayFilter {
+
+    public static void main(String[] args) {
+
+        final String Result = "Результат вашего запроса следующий :";
+        final String OneMore = "Желаете попробовать еще раз? \n1 | да \n2 | нет";
+        Scanner sc = new Scanner(System.in);
+        boolean check = true;
+        byte more;
+        int totalresult;
+
```

Comment:
Please remove extra lines

Reply:
Done. Is it ok to leave one empty line between to different blocks of code ?

Reply:
Better to keep it as short as possible